secure/secure.sh

```diff
@@ -464,14 +589,20 @@ function create_dataproc_principal() {
   chmod 400 /etc/dataproc.keytab
 }
 
+function krb5_propagate() {
+  if [[ "${FQDN}" == "${MASTER_FQDN}" && "${HA_MODE}" == 1 ]]; then
+    /etc/krb5_prop.sh
```

Comment:
Since this is referenced in multiple places, it should be a variable at the top of the file

Reply:
done.